numba/roc/gcn_occupancy.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import division, print_function
+
+import math
+from collections import namedtuple
+
+
+# GCN architecutre specific info
+simd_per_cu = 4
+wave_size = 64
+vector_register_file_size = 64 * 2**10  # 64 kB
+byte_per_VGPR = 4
+vgpr_per_simd = vector_register_file_size // byte_per_VGPR
+sgpr_per_simd = 512
+max_wave_count = 10
+max_inflight_wave_per_cu = max_wave_count * simd_per_cu
+
+# XXX due to limit in AMDGPU backend
+max_group_size = 256
```

Comment:
I recall there maybe changes to this.  Are there?

Reply:
There are, but right now I'd elect to keep this as-is and update it as more information becomes available.